src/libs/Localize/index.ts

```diff
@@ -72,6 +76,58 @@ const translationCache = new Map<ValueOf<typeof CONST.LOCALES>, Map<TranslationP
     }, [] as Array<[ValueOf<typeof CONST.LOCALES>, Map<TranslationPaths, string>]>),
 );
 
+/**
+ * Helper function to handle pluralized translations.
+ *
+ * This function takes the result of a translated phrase function,
+ * along with the language, phrase key, and count, and returns the
+ * appropriate pluralized form of the translation.
+ *
+ * It uses the Intl.PluralRules API to determine the correct plural form
+ * based on the count and the language's pluralization rules.
+ *
+ * If the specific plural form is not found, it falls back to the 'other' form.
+ *
+ * @param result - The result of the translated phrase function.
+ * @param language - The language of the translation.
+ * @param phraseKey - The key of the phrase being translated.
+ * @param count - The count used to determine the plural form as a string but numerical form (e.g. '5').
+ * @param phraseParameters - The parameters to pass to the translated phrase function.
+ * @returns The pluralized form of the translation, or null if not found.
+ */
+function getPluralTranslation<TKey extends TranslationPaths>(
+    result: Record<Intl.LDMLPluralRule, Phrase<TKey>>,
+    language: string,
+    phraseKey: TKey,
+    count: number,
+    phraseParameters?: Record<string, unknown>,
+): string | null {
+    const pluralRules = new Intl.PluralRules(language);
```

Comment:
@brandonhenry I think passing only locale will not be sufficient in `new Intl.PluralRules(language);`; we also need to have passed the `type "ordinal"` `new Intl.PluralRules(language, {
        type: 'ordinal'
    });`. I was testing the count param, and right now, passing one and the other is working only; if we pass two, three, or any number, it will return the other instead of two or three.

Reply:
@brandonhenry Did you get what I am trying to say here ? ⬆️